guides/image-sources/prov-installation-media.iuml

```diff
@@ -0,0 +1,6 @@
+loop
+   Host -> Foreman : requests installation media
+   Foreman -> Proxy : requests installation media
+   Proxy -> Foreman : provides installation media
```

Comment:
I think the call would be "Installation media" because it is in response to a request. Same comment for next line.